dbcon/joblist/tupleaggregatestep.cpp

```diff
@@ -5499,7 +5499,17 @@ void TupleAggregateStep::threadedAggregateRowGroups(uint32_t threadID)
                 rgDatas.clear();
                 fRm->returnMemory(fMemUsage[threadID], fSessionMemLimit);
                 fMemUsage[threadID] = 0;
-
+                if (multiDist)
+                {
+                    for (uint32_t j = 0; j < multiDist->subAggregators().size(); j++)
+                    {
+                        if (distRow[j].usesStringTable() && distRowData[j].getStringTableMemUsage() > 50 * (1 << 20))
```

Comment:
We should move this threshold value (50 * (1 << 20)) into a common location (maybe in rowgroup.h) as a const int or a macro, so it can be easily updated later.

Use of UNLIKELY around the if predicate might be a suitable optimization?